src/app/test/test.component.ts

```diff
@@ -65,7 +67,7 @@ export class TestComponent implements OnInit, AfterViewInit {
   }
 
   ngAfterViewInit(): void {
-    this.loadData(null);
+    this.loadData('');
```

Comment:
Would an empty path mean the root of the project? Because than, I think it could be better to have an extra method called `loadRootData();` with passes the `this.testFileTreeComponent.rootFolder.name` to the `loadData`. It moves some of the logic out of the way, and then the check to see if the path is empty is more meaningful.

Reply:
I'm not sure i understand. Do you mean to call `this.loadRootData()` instead of `this.loadData()` and have `this.loadRootData()` give `loadData()` either the given path or the root path depending on if it has gotten an input?

Reply:
`loadRootData` shouldn't have any parameters and always give the root path to `loadData`. Then `loadData` doesn't need to worry about that anymore, cleaning it up somewhat.

Reply:
Thank you for implementing it. Do you think you can also change every `loadData('')` to `loadRootData()`?

Reply:
of course